Authentication/AuthenticationBootstrapper.swift

```diff
@@ -31,6 +31,8 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
         return sessionService.currentUser.value
     }
 
+    private let frameworkBundle: NSBundle = NSBundle(forClass: LoginView.self)
```

Comment:
I don't like having this in all the classes which have a view. I would rather have (in a separate, common file) a 
```
    internal let FrameworkBundle = NSBundle(forClass: LoginView.self)
```
and then use that one.